### PR TITLE
Add multiple secondary action design guidance to Empty States 

### PR DESCRIPTION
**Pull request by nding-anges**
…on design
Fixed issue #907
Updated Design page of Empty States based on new design solution and [Demo page](https://rawgit.com/dabeng/patternfly/empty-state-dist/dist/tests/blank-slate.html)
Relative issue #696 

## Description
* Add two alternative design of multiple secondary action including design guidance to Empty States Design page. 

## Changes

Write a list of changes this design pattern introduces

* image of design page
* Relative guidance  

## Contribution Submission Checklist

- [X] A GitHub issue was created and reviewed with the PatternFly community.
- [ ] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [ ] Any new files have been added to the folder for that pattern.
- [ ] The folder, for brand new patterns, has been titled with the name of the pattern.
- [X] All images are in an "img" folder and embedded images have the correct filepath.
- [X] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

File: pattern-library/communication/empty-state/design/design.md [hunk 1]
```diff
@@ -2,10 +2,17 @@
...
 
-![Empty State with Callouts](img/empty-state.png)
+![Empty State with Callouts](img/empty-state-new.png)
 
 1. **Icon** (optional): If an object icon is associated with this view, it can be displayed here.
 1. **Empty State Title:** Give it a name.
 1. **Text** (optional): This text should be personal and helpful. It should minimize the user’s effort to complete tasks.
 1. **Primary Action Button:** The primary action is displayed as a prominent blue button. See the [Action Labels](http://www.patternfly.org/styles/terminology-and-wording/#action-labels) section for more information about terminology and wording specific to action labels.
-1. **Secondary Action Buttons** (optional): Secondary actions are alternative options for the user. They are shown as more subtle gray buttons and located below the main action. There can be more than one secondary action.
+1. **Secondary Action Buttons** (optional): Secondary actions are alternative options for the user. They are shown as more subtle gray buttons and located below the main action. There can be more than one secondary action. See more alternative design for multiple secondary action:
```
Comment (mcarrano) [inline pattern-library/communication/empty-state/design/design.md]: Change last sentence to, "Here are some additional variants for displaying secondary actions:"

File: pattern-library/communication/empty-state/design/design.md [hunk 2]
```diff
@@ -2,10 +2,17 @@
...
+
+
+  - If there are not  extremely long action labels for all the secondary action, we recommend  using the secondary gray button style. In case of differing  action label length, user could set an appropriate button width  based on the most longest action label and keep equal width for all secondary buttons.
```
Comment (mcarrano) [inline pattern-library/communication/empty-state/design/design.md]: Remove the first sentence. Change the second sentence to, "In the case of differing action label lengths, set a consistent button width based on the longest action label."

File: pattern-library/communication/empty-state/design/design.md [hunk 3]
```diff
@@ -2,10 +2,17 @@
...
+  ![multiple secondary button style](img/secondary-button-style.png)
+
+  - If there are too much secondary actions or some of action labels are extremely long,  we recommend using text link button style.
```
Comment (mcarrano) [inline pattern-library/communication/empty-state/design/design.md]: Change to, "If there are more than 3 secondary actions, we recommend using text links in place of buttons."

**Comment by nding-anges** [comment]
@mcarrano Thanks for your modifications.

**Pull request merged**